src/main/java/org/infrastructurebuilder/audit/excel/ExcelGeneratorPackageMojo.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright © 2019 admin (admin@infrastructurebuilder.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.infrastructurebuilder.audit.excel;
+
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Component;
+import org.apache.maven.plugins.annotations.LifecyclePhase;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.plugins.annotations.ResolutionScope;
+import org.apache.maven.project.MavenProject;
+import org.apache.maven.project.MavenProjectHelper;
+
+import java.io.File;
+import java.nio.file.Paths;
+
+
+@Mojo(name = "excel-generator", defaultPhase = LifecyclePhase.PACKAGE, requiresDependencyResolution = ResolutionScope.RUNTIME, requiresProject = true, threadSafe = true)
+public class ExcelGeneratorPackageMojo extends AbstractMojo {
+
+  @Parameter(defaultValue = "${project.build.directory}/", readonly = true)
+  private File workDirectory;
+
+  @Parameter(defaultValue = "${project.artifactId}-${project.version}-excel.xlsx", readonly = true)
+  private String outputFile;
+
+  @Parameter(property = "project", readonly = true, required = true)
+  private MavenProject project;
+
+  @Parameter(property = "classifier")
+  private String classifier;
+
+  @Component
+  private MavenProjectHelper mavenProjectHelper;
+
+  @Override
+  public void execute() throws MojoExecutionException, MojoFailureException {
+    if (!getPluginContext().containsKey(ExcelGeneratorTestMojo.EXCEL_RESULT_XLSX)) {
```

Comment:
Remove this as part of the migration (you won't need `EXCEL_RESULT_XLSX` anymore since this will all happen inside the one class)